src/components/Button.jsx

```diff
@@ -0,0 +1,14 @@
+import React from "react";
+
+function Button({ color, onClick,children}) {
```

Comment:
`const { theme } = useContext(ThemeContext);` should be used here to control the foreground and background